packages/functional-tests/tests/syncV3/fxDesktopHandshake.spec.ts

```diff
@@ -0,0 +1,274 @@
+    target,
+  }) => {
+    const { login, page } = syncBrowserPages;
+    const query = new URLSearchParams({
+      forceUA: uaStrings['desktop_firefox_71'],
+      email: otherEmail,
+    });
+    const eventDetailStatus = createCustomEventDetail(
+      FirefoxCommand.FxAStatus,
+      {
+        signedInUser: browserSignedInAccount,
+      }
+    );
+    await page.goto(
+      `${
+        target.contentServerUrl
+      }?force_auth&automatedBrowser=true&context=fx_desktop_v3&service=sync&${query.toString()}`
+    );
+    await login.respondToWebChannelMessage(eventDetailStatus);
+    await login.checkWebChannelMessage('fxaccounts:fxa_status');
+    expect(await login.getPrefilledEmail()).toContain(otherEmail);
+  });
+
+  test('Non-Sync force_auth page - user signed into browser is different to requested user', async ({
+    target,
+  }) => {
+    const { login, page } = syncBrowserPages;
+    const query = new URLSearchParams({
+      forceUA: uaStrings['desktop_firefox_71'],
+      email: otherEmail,
+    });
+    const eventDetailStatus = createCustomEventDetail(
+      FirefoxCommand.FxAStatus,
+      {
+        signedInUser: browserSignedInAccount,
+      }
+    );
+    await page.goto(
+      `${
+        target.contentServerUrl
+      }?force_auth&automatedBrowser=true&${query.toString()}`
+    );
+    await login.respondToWebChannelMessage(eventDetailStatus);
+    await login.checkWebChannelMessage('fxaccounts:fxa_status');
+    expect(await login.getPrefilledEmail()).toContain(otherEmail);
+  });
+
+  test('Sync - user signed into browser, no user signed in locally', async ({
+    target,
+  }) => {
+    const { login, page } = syncBrowserPages;
+    const query = new URLSearchParams({
+      forceUA: uaStrings['desktop_firefox_71'],
+      email: browserSignedInEmail,
+    });
+    const eventDetailStatus = createCustomEventDetail(
+      FirefoxCommand.FxAStatus,
+      {
+        signedInUser: browserSignedInEmail,
+      }
+    );
+    await page.goto(
+      `${
+        target.contentServerUrl
+      }?context=fx_desktop_v3&service=sync&automatedBrowser=true&${query.toString()}`
+    );
+    await login.respondToWebChannelMessage(eventDetailStatus);
+    await login.checkWebChannelMessage('fxaccounts:fxa_status');
+    expect(await login.getPrefilledEmail()).toContain(browserSignedInEmail);
+  });
+
+  test('Non-Sync - user signed into browser, no user signed in locally', async ({
+    target,
+  }) => {
+    const { login, page } = syncBrowserPages;
+    const query = new URLSearchParams({
+      forceUA: uaStrings['desktop_firefox_71'],
+      email: browserSignedInEmail,
+    });
+    const eventDetailStatus = createCustomEventDetail(
+      FirefoxCommand.FxAStatus,
+      {
+        signedInUser: browserSignedInEmail,
+      }
+    );
+    await page.goto(
+      `${target.contentServerUrl}?/automatedBrowser=true&${query.toString()}`
```

Comment:
See previous comment above re: suspicious `?/automatedBrowser=` and how this _should_ be creating a query param with a literal forward slash in the param name.

```suggestion
      `${target.contentServerUrl}?automatedBrowser=true&${query.toString()}`
```

Reply:
Thanks, made the change.